src/storage/settings.ts

```diff
@@ -50,6 +52,28 @@ export const foodLogTotalsPositionAtom = atomWithStorage<
   getOnInit: true,
 });
 
+export const foodLogGoalsPositionAtom = atomWithStorage<
+    "hidden" | "top" | "bottom" | "both"
+>("macro-goals:position", "top", undefined, {
```

Comment:
One thought is that maybe we should default to "hidden" since I think most people track calories rather than macros.

Reply:
Done